lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -1451,6 +1451,8 @@ namespace Js
 
     RecyclableObject* JavascriptProxy::GetPrototypeSpecial()
     {
+        PROBE_STACK(GetScriptContext(), Js::Constants::MinStackDefault);
```

Comment:
I have a test but that is something exhausting the stack. Pretty much revealing that probe_stack is missing. I don't see a value in it. What do you think?